diff --git a/rpmconf/rpmconf.py b/rpmconf/rpmconf.py
index 20fefff..9b374c8 100755
--- a/rpmconf/rpmconf.py
+++ b/rpmconf/rpmconf.py
@@ -244,11 +244,11 @@ class RpmConf(object):
     def _ls_conf_file(self, conf_file, other_file):
         print("Configuration file '{}'".format(conf_file))
         if self.selinux:
-            print(subprocess.check_output(['/usr/bin/ls', '-ltrd', '--context',
+            print(subprocess.check_output(['/bin/ls', '-ltrd', '--context',
                                            conf_file, other_file],
                                           universal_newlines=True))
         else:
-            print(subprocess.check_output(['/usr/bin/ls', '-ltrd',
+            print(subprocess.check_output(['/bin/ls', '-ltrd',
                                            conf_file, other_file],
                                           universal_newlines=True))
 
